source/draw/gpu/opengl/backend/gles3/Gles3FramebufferObject.ooc

```diff
@@ -26,39 +26,39 @@ version(!gpuOff) {
 Gles3FramebufferObject: class extends GLFramebufferObject {
 	init: func (=_size) { super() }
 	free: override func {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject free") }
 		glDeleteFramebuffers(1, _backend&)
 		version(debugGL) { validateEnd("FramebufferObject free") }
 		super()
 	}
 	bind: func {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject bind") }
 		glBindFramebuffer(GL_FRAMEBUFFER, this _backend)
 		version(debugGL) { validateEnd("FramebufferObject bind") }
 	}
 	unbind: func {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject unbind") }
 		glBindFramebuffer(GL_FRAMEBUFFER, 0)
 		version(debugGL) { validateEnd("FramebufferObject unbind") }
 	}
 	scissor: static func (x, y, width, height: Int) {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject scissor") }
 		glScissor(x, y, width, height)
 		version(debugGL) { validateEnd("FramebufferObject scissor") }
 	}
 	clear: static func {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject clear") }
 		glClear(GL_COLOR_BUFFER_BIT)
 		version(debugGL) { validateEnd("FramebufferObject clear") }
 	}
 	setClearColor: func (color: ColorBgra) {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("FramebufferObject setClearColor") }
 		normalized := color normalized
 		glClearColor(normalized x, normalized y, normalized z, normalized w)
 		version(debugGL) { validateEnd("FramebufferObject setClearColor") }
 	}
 	readPixels: func -> ByteBuffer {
-		version(debugGL) { validateStart() }
+		version(debugGL) { validateStart("fbo readPixels") }
```

Comment:
You use `FramebufferObject` in most places but `fbo` in others. Are they different somehow? Otherwise, be consistent.